src/frontends/pytorch/src/node_context.cpp

```diff
@@ -228,6 +228,8 @@ Any NodeContext::get_values_from_const_input(int index) const {
             return get_constant_data<int16_t>(constant);
         case element::u16:
             return get_constant_data<uint16_t>(constant);
+        case element::boolean:
+            return get_constant_data<bool>(constant);
```

Comment:
Why do we need it for docs? Can we remove it? Leaving it here means we cannot merge it in the release branch.

Reply:
We can have it separately for this PR, however I can see that the docs update for release branch was already merged.